docs/conf.py

```diff
@@ -117,6 +119,8 @@
     (r"py:.*", "baybe.constraints.conditions.Condition.__init__"),
     (r"py:.*", "baybe.utils.serialization.SerialMixin.__init__"),
     (r"DeprecationWarning:", ""),
+    # Ignore the generics in utils.basic
+    (r"py:class", "baybe.utils.basic._U"),
```

Comment:
what about the other one `_T` ? 
<img width="188" alt="image" src="https://github.com/emdgroup/baybe/assets/17951239/191d110b-ba4a-4e02-8908-8d533d8d0afa">

Reply:
It does not complain about that (which I do not understand - only explanation that I have is that this is such a standard choice of variable name that this is checked explicitly somewhere?)
Could however add that as well for "full transparency" resp. see whether I can come up with a suitable regex here.

Reply:
Ok, I suggest simply add the T as well, this seems the most robust version.